src/app/load-app.js

```diff
@@ -9,37 +9,42 @@ import css from './load-app.module.css'
 const LoadApp = ({ children }) => {
     useCustomFormsPrefetch()
 
-    const { isLoading, isError, data, error } = useMetadata()
-    const {
-        isLoading: userLoading,
-        isError: userIsError,
-        data: userData,
-    } = useUserInfo()
-
-    if (isLoading || userLoading) {
+    const metadata = useMetadata()
+    const userInfo = useUserInfo()
+
+    if (metadata.isLoading || userInfo.isLoading) {
         return (
             <CenteredContent className={css.center} position={'top'}>
                 <CircularLoader />
             </CenteredContent>
         )
     }
 
-    if (isError) {
-        console.error(data)
+    if (metadata.isError) {
         return (
             <NoticeBox
-                className={css.noticeBoxWrapper}
                 error
+                className={css.noticeBoxWrapper}
                 title={i18n.t('There was a problem loading metadata')}
             >
-                {error?.message}
+                {metadata.error}
```

Comment:
Isn't error and object? And wouldn't we want to render a string?

Reply:
Also, what about the `userInfo.isError` case? Don't we need to show an error notification for that too? I expect that other parts of the app are relying on the user info to be there.

Reply:
See comment below